i18n/locale-info.php

```diff
@@ -0,0 +1,6814 @@
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'en-CC' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'en-CX' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'en-KI' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'en-NF' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'en-NR' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'en-TV' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'ki'    =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+	],
+	'AWG' =>
+	[
+		'nl-AW' =>
+		[
+			'currency_pos' => 'left_space',
+			'thousand_sep' => '.',
+			'decimal_sep'  => ',',
+		],
+	],
+	'AZN' =>
+	[
+		'az' =>
+		[
+			'currency_pos' => 'right_space',
+			'thousand_sep' => '.',
+			'decimal_sep'  => ',',
+		],
+	],
+	'BAM' =>
+	[
+		'hr-BA'      =>
+		[
+			'currency_pos' => 'right_space',
+			'thousand_sep' => '.',
+			'decimal_sep'  => ',',
+		],
+		'sr-Cyrl-BA' =>
+		[
+			'currency_pos' => 'right_space',
+			'thousand_sep' => '.',
+			'decimal_sep'  => ',',
+		],
+		'sr-Latn-BA' =>
+		[
+			'currency_pos' => 'right_space',
+			'thousand_sep' => '.',
+			'decimal_sep'  => ',',
+		],
+	],
+	'BBD' =>
+	[
+		'en-BB' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+	],
+	'BDT' =>
+	[
+		'bn' =>
+		[
+			'currency_pos' => 'right',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+	],
+	'BGN' =>
+	[
+		'bg' =>
+		[
+			'currency_pos' => 'right_space',
+			'thousand_sep' => ' ',
+			'decimal_sep'  => ',',
+		],
+	],
+	'BHD' =>
+	[
+		'ar-BH' =>
+		[
+			'currency_pos' => 'right_space',
+			'thousand_sep' => '٬',
+			'decimal_sep'  => '٫',
+		],
+	],
+	'BIF' =>
+	[
+		'en-BI' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'fr-BI' =>
+		[
+			'currency_pos' => 'right_space',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+	],
+	'BMD' =>
+	[
+		'bm'    =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'en-BM' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+	],
+	'BND' =>
+	[
+		'ms-BN' =>
+		[
+			'currency_pos' => 'left_space',
+			'thousand_sep' => '.',
+			'decimal_sep'  => ',',
+		],
+	],
+	'BOB' =>
+	[
+		'bo'    =>
+		[
+			'currency_pos' => 'left_space',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'es-BO' =>
+		[
+			'currency_pos' => 'left',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+		'qu-BO' =>
+		[
+			'currency_pos' => 'left_space',
+			'thousand_sep' => ',',
+			'decimal_sep'  => '.',
+		],
+	],
+	'BRL' =>
```

Comment:
I know this is not yet ready for review, but the thousand separator for the Brazilian real should be `.` instead of ` `.

I'm not sure what happened here, but from what I could see, the `br` locale is for [Breton](https://www.localeplanet.com/icu/br/index.html). The default Brazil's locale should be [`pt-BR`](https://www.localeplanet.com/icu/pt-BR/index.html), or in the CLDR's case [just `pt`](http://cldr.unicode.org/translation/translation-guide-general/default-content): *"Portuguese (Brazil) [pt_BR] is the default content locale for Portuguese [pt]"*.

I've also tried the CLDR Survey tool for [`es-BR`](https://st.unicode.org/cldr-apps/v#/es_BR/Number_Formatting_Patterns/) and, similarly to [`pt`](https://st.unicode.org/cldr-apps/v#/pt/Number_Formatting_Patterns/), it should be using `.`.

Reply:
Oops, then my matching locales with country codes to find a default one algorithm has some issues. I took this path:

1. If there's any locales assigned to that country in CLDR, return empty array
2. If there's one, return that
3. check for "XX" == country code, if there's one, return it
3. check for "XX-{country code}", if there's only one, return it
4. check for "{country-code}-XX", if there's only one, return it
5. group locale data by their configurations, which configuration has more occurrences, return the first of it

then `pt-BR` (4th check) should be before `br` (3rd check). Or maybe I'll find something more reliable.

Reply:
There's no `pt-BR` locale defined in the CLDR, and there's an issue about it, they say `pt` is the default locale for Brazilian Portuguese. Interesting. I need to map it by hand then. https://github.com/elixir-cldr/cldr/issues/42